Tasks/Maksim/Task3.1/query09.sql

```diff
@@ -0,0 +1,20 @@
+--Query  9
+SELECT customer.CustomerID,
+       SOH.OrderDate,
+       CASE
```

Comment:
look at `coalesce`/`isnull`
so it would be no reason to create such a complecated construction with case